src/components/StyledTripDetails.js

```diff
@@ -0,0 +1,69 @@
+import styled from "styled-components";
+
+export const StyleButton = styled.button`
+  display: flex;
+  position: fixed;
+  margin-top: 6rem;
+  margin-left: 0.5rem;
+  border-radius: 5px;
+  background-color: #0d5c63;
+  color: yellow;
+  bottom: 1rem;
+  font-size: 1rem;
+`;
+export const StyleSpan = styled.span`
+  font-family: "Lobster", cursive;
+  display: inline;
+  margin-left: 20px;
+  @media screen and (max-width: 768px) {
+    margin-right: 1rem;
+    font-size: 18px;
```

Comment:
I'm not a fan of using px for font-size, but this is also something to keep in mind for refactoring rather than something that needs to be changed now.